appveyor.yml

```diff
@@ -115,22 +111,23 @@ install:
   # done from inside the powershell script as it would require to restart
   # the parent CMD process).
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "SET PIP_DISABLE_PIP_STRING_CHECK=1"
```

Comment:
My bad, that should be `SET PIP_DISABLE_PIP_VERSION_CHECK=1`

Reply:
https://github.com/biosustain/swiglpk/pull/43/commits/9b5c5b83f83c7bc7932e632d731006e41037d15a